src/daemon/instance_settings_handler.cpp

```diff
@@ -216,13 +261,20 @@ void mp::InstanceSettingsHandler::set(const QString& key, const QString& val)
         update_cpus(key, val, instance, spec);
     else
     {
-        auto size = get_memory_size(key, val);
-        if (property == mem_suffix)
-            update_mem(key, val, instance, spec, size);
+        if (property == bridged_suffix)
```

Comment:
`else if`? To avoid one extra nesting level.

Reply:
Ok, changed. I was hesitating whether to mix an inline `else if` with a structured `else { stmt; if ... }`. But right, we save four columns the way you mention.

Reply:
It's not for saving spaces, it's that it makes it easier to understand that they are alternatives at the same level, with no further shared logic (unlike the following 2 properties, which do share the conversion of the memory size).